guide.css

```diff
@@ -35,6 +35,10 @@ img{
 	display: inline;
 	overflow: hidden;
 }
+.collapsibleContent a:not(span *){
+	color: black;
+	text-decoration: underline dotted grey; 
+}
```

Comment:
why are we changing all a links? didn't we want that only for glitches?

Reply:
I'm personally not a fan of the standard blue hyperlink formatting within the guide instructions, and aside from a few map links (which are redundant anyway) and the TOC, all the links selected by this css are glitches anyway. Probably easier at this point to fix it for the TOC using css rather than manually fixing every glitch link in the entire v4 guide to have a class that can be edited with css, and then once we get to the phase of making a better parser for next time I'll make the parser output include a class="glitch" or something and fix it then.